src/utils/offer/OfferUtils.js

```diff
@@ -20,5 +20,5 @@ export const validApplyURL = (val) => {
     const httpRegex = /^https?:\/\/\S+\.\S+$/;
     const emailRegex = /^mailto:(\S+@\S+)$/;
 
-    return httpRegex.test(val) || emailRegex.test(val);
+    return httpRegex.test(val) || emailRegex.test(val) || (val === "") || (val === null);
```

Comment:
Why does the `(val === "")` need to be allowed?
And what about the `(val === null)`? Is yup's validation in https://github.com/NIAEFEUP/nijobs-fe/blob/7ca25fac86ef595b7e4976ded0ffa1504c7ca18d/src/components/Offers/Edit/EditOfferSchema.js#L76 not enough?

Reply:
You cannot submit the edit offer form because it expects the value to start with _https_ or _mailto:_, therefore, in order to make it optional it should accept the "" value. This happens when you edit the offer the first time, the second time, it fetches the value which is null, so it should accept the null value too, in case the user submits without changing the value.

Reply:
Have you looked further into this ?